content/foundations/icons/index.mdx

```diff
@@ -6,6 +6,11 @@ source: https://github.com/primer/octicons
 import Icons from '../../../src/components/icons'
 import {HeartFillIcon} from '@primer/octicons-react'
 
-A scalable set of icons handcrafted with <HeartFillIcon /> by GitHub
+Primer Octicons are GitHub’s open source library of icons for GitHub UI. These are visual metaphors intended to serve as a visual language. If you are looking for brand and marketing logos and visuals, see <a href="/foundations/icons/octovisuals">Octovisuals</a>.
+
+<a href="/foundations/icons/design-guidelines">See Design and usage guidelines</a>
```

Comment:
Can we use markdown links here?

```suggestion
Primer Octicons are GitHub’s open source library of icons for GitHub UI. These are visual metaphors intended to serve as a visual language. If you are looking for brand and marketing logos and visuals, see [Octovisuals](/foundations/icons/octovisuals).

[See Design and usage guidelines](/foundations/icons/design-guidelines)
```

Reply:
@JoshBowdenConcepts I was just cleaning up the PRs and saw that this one is pending your input 🙂